modules/local/blobtoolkit/create_dataset/environment.yml

```diff
@@ -0,0 +1,8 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - conda-forge::python=3.9
+  - conda-forge::blobtoolkit=4.3.9
```

Comment:
Is this correct? A quick search suggests no BTK package in conda-forge, but one in bioconda (v4.4.6 only).

Reply:
Good catch, this may have been a placeholder that i meant to come back to.

sanger-tol/btk is using
```
    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
        exit 1, "BLOBTOOLKIT_BLOBDIR module does not support Conda. Please use Docker / Singularity / Podman instead."
    }
    container "docker.io/genomehubs/blobtoolkit:4.4.6"
```

So i'll default to that.